<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serverItem</class>
 <widget class="QWidget" name="serverItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>46</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="server_button">
       <property name="statusTip">
        <string>Connectar al servidor</string>
       </property>
       <property name="text">
        <string>Servidor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="time_progress">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="space_progress">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
